src/firebase/firebase.ts

```diff
@@ -184,11 +185,12 @@ export function* initializeCloudMessaging(app: ReactNativeFirebase.Module, addre
       yield put(pushNotificationsPermissionChanged(pushNotificationsEnabled))
     }
   }
-
-  // `registerDeviceForRemoteMessages` must be called before calling `getToken`
-  // Note: `registerDeviceForRemoteMessages` is really only required for iOS and is a no-op on Android
-  yield call([app.messaging(), 'registerDeviceForRemoteMessages'])
```

Comment:
this isn't needed anymore?

Reply:
Not anymore now that we've turned off [messaging_ios_auto_register_for_remote_messages](https://github.com/valora-inc/wallet/pull/3631/files#r1173119135)

docs here https://rnfirebase.io/app/json-config#messaging_ios_auto_register_for_remote_messages

Reply:
Removing this + `messaging_ios_auto_register_for_remote_messages` will cause iOS to show the push permission popup directly on app launch, which is a bit too intrusive I think.

Are we sure we want this?